<?xml version="1.0"?>
<launch>
  <node name="ros_jpp_pointcloud_node" pkg="ros_jpp" type="ros_jpp_pointcloud_node" output="screen">
    <param name="max_pts" value="4000"/>
    <!--remap from="velodyne_points" to="velodyne_top_driver/velodyne_points"/-->
  </node>
</launch>
